jep/1/README.adoc

```diff
@@ -152,6 +152,7 @@ A JEP may have any number of "Contributors" who
 help write, implement, discuss, or offer feedback about that JEP.
 One contributor might do only one or any combination of these
 of these things during any part of the life of a JEP.
+Companies or legal entitites cannot be contributors.
```

Comment:
I think I mirror @oleg-nenashev's concerns here. This is off-putting for corporate contribution IMHO.

Reply:
This was the most straightforward way to prevent something "Sponsor: Praqma" in #31. I'll probably not email info@praqma.com, or ping @Praqma, about concerns about a JEP.

Would it help to rephrase this to something like the following?

> While we invite contributions by companies or other organizations, contributors (such as sponsors or BDFL Delegates below) need to be specific people.

Reply:
I think that's much better, yes.

(Soylent) Jenkins is made of people!

Reply:
@daniel-beck Works for me, 👍 
We could also allow mentioning affiliation directly in JEP. E.g. `[Oleg Nenashev](GitHub link), [CloudBees, Inc.](Website link)` in the doc